setup.py

```diff
@@ -24,6 +24,7 @@
     install_requires=[
         'Markdown==2.6.2',
         'bleach==1.4.2',
+        'Pygments==2.1.3',
```

Comment:
if "Syntax highlighting using codehilite is an optional feature", shouldn't Pygments be an optional requirement too?